Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -642,15 +656,10 @@ alias snorm_norm := eLpNorm_norm
 
 theorem eLpNorm'_norm_rpow (f : α → F) (p q : ℝ) (hq_pos : 0 < q) :
     eLpNorm' (fun x => ‖f x‖ ^ q) p μ = eLpNorm' f (p * q) μ ^ q := by
-  simp_rw [eLpNorm']
-  rw [← ENNReal.rpow_mul, ← one_div_mul_one_div]
-  simp_rw [one_div]
-  rw [mul_assoc, inv_mul_cancel₀ hq_pos.ne.symm, mul_one]
-  congr
-  ext1 x
-  simp_rw [← ofReal_norm_eq_coe_nnnorm]
-  rw [Real.norm_eq_abs, abs_eq_self.mpr (Real.rpow_nonneg (norm_nonneg _) _), mul_comm, ←
-    ENNReal.ofReal_rpow_of_nonneg (norm_nonneg _) hq_pos.le, ENNReal.rpow_mul]
+  simp_rw [eLpNorm'_eq_lintegral_nnnorm, ← ENNReal.rpow_mul, ← one_div_mul_one_div, one_div,
```

Comment:
This golf was also mostly pre-existing, right? (I'll happily review the PR with just this golf. Not required from my side.)